### PR TITLE
Incrementing README.md with Jest usage.

### PR DESCRIPTION
**Pull request by diogocezar**
Just inserted new section explaining how we can use this package with Jest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,21 @@ module.exports = {
 }
 ```
 
+## Usage with Jest
+
+If you use jest to test your application, you can structure this kind of configuration to run tests:
```
Comment (ilearnio) [inline README.md]:
```suggestion
Unfortunately, `module-alias` itself would not work from Jest due to a custom behavior of Jest's `require`. But you can use it's own aliasing mechanism instead. The configuration can be defined either in `package.json` or `jest.config.js`. The example below is for `package.json`:
```

File: README.md [hunk 2]
```diff
@@ -143,6 +143,21 @@ module.exports = {
...
+```js
+"jest": {
+  "moduleNameMapper": {
+    "@root/(.*)": "<rootDir>/$1",
+    "@client/(.*)": "<rootDir>/src/client/$1"
+  },
+}
+```
+
+Just put it on `package.json` or in yout `jest.config.js`
```
Comment (ilearnio) [inline README.md]:
```suggestion
```

**Comment by diogocezar** [comment]
For me, sounds good ;)

**Comment by ilearnio** [comment]
Thank you! Merging it

**Pull request merged**